'

```diff
@@ -0,0 +1,10 @@
+[AVR] Uncommented basic block printing statements
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+# On branch avr-support
+# Your branch is ahead of 'origin/avr-support' by 7 commits.
+#   (use "git push" to publish your local commits)
+#
+# Changes to be committed:
+#	modified:   lib/Target/AVR/AVRAsmPrinter.cpp
+#
```

Comment:
Yo dawg! I commited the commit message... :-D

File is called "'" (single quote)...